docs/static/releasenotes.asciidoc

```diff
@@ -72,7 +72,6 @@ This section summarizes the changes in the following releases:
 ==== Notable issues fixed
 * Fix a possible corruption of Persistent Queue during a crash of the Logstash process https://github.com/elastic/logstash/pull/14147[#14147]
 * Fixes to the pre-flight check for starting a pipeline with the Persistent Queue:
-** We correctly validate free capacity on the _volume_ on which the pipeline's PQ is configured https://github.com/elastic/logstash/pull/14212[#14212]
```

Comment:
is this intentional?

Reply:
yes; the relevant PR was _tagged_ with 7.17.5, but was not merged prior to the BC for 7.17.5 and didn't make it into the release.